modules/ROOT/pages/_partials/sync/js-sync-js-basic-concepts.adoc

```diff
@@ -1,8 +1,12 @@
-= Basic concepts
+= Introduction
 
-Client is basing on Apollo GraphQL client that can be used with various web and mobile frameworks.
-We provide version for web and Apache Cordova.
-For basic concepts about Apollo GraphQL please refer to documentation for your own platform.
+The {sync-service} Javascript client SDK is built on top of the popular link:https://www.apollographql.com/docs/react[Apollo GraphQL Javascript client], and it adds the following additional features: 
+
+* Offline query and mutation support
+* Conflict detection and resolution
+* Audit logging
+
+The client SDK can be used with all the major javascript frameworks that Apollo supports, please refer to their documentation for some of the basic concepts for each of the frameworks:
```

Comment:
Do we want to recommend React?
This seems vague, what works?, what's tested?
Eg 
The {data-client-sdk} is tested with React, but may work with other frameworks that Apollo supports. Please ..

Reply:
good question. @wtrocki did we actually test Voyager client against Apollo React or Apollo Angular?

Reply:
apollo-client is an js library. We extend apollo client. On top of that there are react and angular packages that provide framework specific bindings. Users have choice to use react/angular/plain js aproach.  Testing this sounds strange as we do not change anything in any of the packages etc. We are just adding links to the request chain.

Reply:
@wtrocki ok, then I will make it clear that the voyager client is an extension of the Apollo client and it can be used with all the Apollo view layers

Reply:
Because the {data-client-sdk} is an extension to Apollo, it works with all the frameworks that Apollo supports.